tests/getEvlTestCertificates.unitTest.ts

```diff
@@ -0,0 +1,43 @@
+import { expect } from "chai";
+import {EvlTestCertificatesService} from "../src/services/EvlTestCertificatesService";
+import fs from "fs";
+import path from "path";
+import { IEvlTestCertificate } from "../@Types/IEvlTestCertificate";
+
+describe("getEvlTestCertificates", () => {
+  let evlTestCertificatesService: EvlTestCertificatesService | any;
+  let MockTestResultsDAO: jest.Mock;
+  let evlTestCertificatesMockDB: any;
+  beforeEach(() => {
+    evlTestCertificatesMockDB = JSON.parse(fs.readFileSync(path.resolve(__dirname, "./resources/evl-test-certificates.json"), "utf8"));
+    MockTestResultsDAO = jest.fn().mockImplementation(() => {
+      return {};
+    });
+    evlTestCertificatesService = new EvlTestCertificatesService(new MockTestResultsDAO());
+  });
+  afterEach(() => {
+    evlTestCertificatesMockDB = null;
+    evlTestCertificatesService = null;
+    MockTestResultsDAO.mockReset();
+  });
+
+  context("when records are found", () => {
+    it("should return a populated response and status code 200", () => {
+      MockTestResultsDAO = jest.fn().mockImplementation(() => {
+        return {
+          getEvlTestCertificates: () => {
+            return Promise.resolve(evlTestCertificatesMockDB);
+          }
+        };
+      });
+
+      evlTestCertificatesService = new EvlTestCertificatesService(new MockTestResultsDAO());
+      return evlTestCertificatesService.getEvlTestCertificates()
+        .then((returnedRecords: IEvlTestCertificate[]) => {
```

Comment:
This should also be async, please follow the guidelines from CVSB-7743 on tech records 

https://github.com/dvsa/cvs-svc-technical-records/blob/feature/CVSB-7743/tests/unit/TechRecordsService.unitTest.ts

Reply:
async await added to the unit test

Reply:
when you are using async/await there is no need for ".then". You are waiting for the Promise to be resolved. The code should look like this:

const returnedRecords = await evlTestCertificatesService.getEvlTestCertificates();
expect(returnedRecords).to.not.equal(undefined);
expect(returnedRecords).to.not.equal({});
....etc